packages/components/SelectionDropdown.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useRef } from "react";
+import { TouchableOpacity, View, ViewStyle } from "react-native";
+
+import chevronDownSVG from "./../../assets/icons/chevron-down.svg";
+import chevronUpSVG from "./../../assets/icons/chevron-up.svg";
+import { BrandText } from "./BrandText";
+import { SVG } from "./SVG";
+import { PrimaryBox } from "./boxes/PrimaryBox";
+import { TertiaryBox } from "./boxes/TertiaryBox";
+import { Separator } from "./separators/Separator";
+import { SpacerColumn } from "./spacer";
+import { useDropdowns } from "../context/DropdownsProvider";
+import {
+  additionalRed,
+  neutral17,
+  neutral44,
+  neutral55,
+  neutral77,
+  secondaryColor,
+} from "../utils/style/colors";
+import { fontMedium14, fontSemibold14 } from "../utils/style/fonts";
+import { layout } from "../utils/style/layout";
+
+interface SelectionDropdownProps {
+  style?: ViewStyle;
+  onDropdownClosed?: () => void;
+  dropdownOptions: string[];
+  placeHolder?: string;
+  setItem: (item: string) => void;
+  item?: string;
+  label?: string;
+}
+
+export const SelectionDropdown = ({
```

Comment:
Same for here, this component is 150 lines long, and it looks like the multiple one, maybe you can find a way to gather them ? We can discuss about it if you want 👍

Reply:
I find the way, but if you have any suggestions, then please let me know.